lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -62,17 +62,19 @@ function getNodesAndTimingByUrl(nodeTimings) {
 }
 
 /**
- *
  * @param {Node} node
  * @param {LH.Gatherer.Simulation.NodeTiming} nodeTiming
  * @param {LH.Artifacts.DetectedStack[]} Stacks
  */
 function computeStackSpecificTiming(node, nodeTiming, Stacks) {
-  let stackSpecificTiming = Object.assign({}, nodeTiming);
-  if(!Stacks) return stackSpecificTiming;
+  const stackSpecificTiming = Object.assign({}, nodeTiming);
```

Comment:
nit: prefer object spreading.

```suggestion
  const stackSpecificTiming = {...nodeTiming};
```